docs/workflows.md

```diff
@@ -1 +1,94 @@
-TODO
\ No newline at end of file
+# Workflows
+
+The purpose of this section of the Kn documentation is to list what we believe are common workflows or use-cases for the Knative CLI. Obviously, as we release the CLI in the wild and get users in production, we might discover novel workflows. As we do we will improve this document.
```

Comment:
s/Obviously....end of paragraph/This is a live document, meant to be updated as we learn more about good ways to use `kn`/

Reply:
More concise. I like it. Thx